css/style.css

```diff
@@ -0,0 +1,158 @@
+/* Fonts */
+@import url("https://fonts.googleapis.com/css?family=Open+Sans|Roboto:400,700&display=swap");
+
+/* Variables */
+
+:root {
+  --primary-1: hsl(205, 86%, 17%);
+  --primary-2: hsl(205, 77%, 27%);
+  --primary-3: hsl(205, 72%, 37%);
+  --primary-4: hsl(205, 63%, 48%);
+
+  /* primary/main color */
+  --primary-5: hsl(205, 78%, 60%);
+
+  /* lighter shades of primary color */
+  --primary-6: hsl(205, 89%, 70%);
```

Comment:
Please use comments as little as possible in the CSS file as using many comments make the page messy.